packages/itinerary-body/src/defaults/line-column-content.tsx

```diff
@@ -1,12 +1,50 @@
 import LocationIcon from "@opentripplanner/location-icon";
 import React, { ReactElement } from "react";
-import { useIntl } from "react-intl";
+import { IntlShape, useIntl } from "react-intl";
 
 import RouteBadge from "../RouteBadge";
 import * as S from "../styled";
 import { LineColumnContentProps } from "../types";
 import { defaultMessages } from "../util";
 
+/**
+ * Gets the travel mode in the ambient language.
+ */
+function getTravelMode(modeId: string, intl: IntlShape): string {
+  switch (modeId) {
+    case "BICYCLE":
+      return intl.formatMessage({
+        defaultMessage:
+          defaultMessages["otpUi.ItineraryBody.travelByMode.bike"],
+        description: "Travel by bike",
+        id: "otpUi.ItineraryBody.travelByMode.bike"
+      });
+    case "CAR":
+      return intl.formatMessage({
+        defaultMessage: defaultMessages["otpUi.ItineraryBody.travelByMode.car"],
+        description: "Travel by car",
+        id: "otpUi.ItineraryBody.travelByMode.car"
+      });
+    case "MICROMOBILITY":
+    case "SCOOTER":
+      return intl.formatMessage({
+        defaultMessage:
+          defaultMessages["otpUi.ItineraryBody.travelByMode.escooter"],
+        description: "Travel by e-scooter",
+        id: "otpUi.ItineraryBody.travelByMode.escooter"
+      });
+    case "WALK":
+      return intl.formatMessage({
+        defaultMessage:
+          defaultMessages["otpUi.ItineraryBody.travelByMode.walk"],
+        description: "Travel by walking",
+        id: "otpUi.ItineraryBody.travelByMode.walk"
+      });
+    default:
+      return modeId;
+  }
+}
```

Comment:
Confirming this is different from the other switch in the access leg?

Reply:
Yes, unfortunately this one has to be separate from the vehicle type because it contains a preposition that can change with the mode (see the French entries for instance).